### ISSUE TITLE
Correct permissions /tmp and /etc

### ISSUE THREAD

**Issue by heichblatt** [archimg/archlinux#6]
Hello,

some days ago, the permissions of `/tmp` changed from `1777` to `0775`, which is incorrect. Please change it back.

Cheers

**Comment by heichblatt** [comment]
Also, shouldn't the permissions of `/etc` be `755`?

From [a build in my Docker Hub repo](https://hub.docker.com/r/heichblatt/archlinux-yaourt/builds/bntgvnznueusoupcptqumkd/) which is based on your image:
```
warning: directory permissions differ on /etc/
filesystem: 775  package: 755
```

**Comment by bebehei** [comment]
Thank you. That's very interesting. I don't know, where I should suspect the bug.

- On my machine (ArchLinux-host) it has `755` for `/etc` and `777` for /tmp. (For /tmp it's ok, but not t-bit is missing.
- Built on a Ubuntu 14.04-host, it has correct permissions and even t-bit set on `/tmp`.
- Built by travis it's `/tmp/` unusable.

**Comment by bebehei** [comment]
```
[bebe:~/code/archimg-archlinux] master ± docker run -it --rm base/archlinux find / -mount -printf '%p %m\n' | sort > findlog-base-archlinux
[bebe:~/code/archimg-archlinux] master ± docker run -it --rm archimg-base-root find / -mount -printf '%p %m\n' | sort > findlog-archimg-root
[bebe:~/code/archimg-archlinux] master ± diff findlog-*
6c6
< /etc 755
---
> /etc 775
548c548
< /etc/pacman.d 755
---
> /etc/pacman.d 775
600
565c565
< /etc/pacman.d/mirrorlist 644
---
> /etc/pacman.d/mirrorlist 664
1166c1166
< /tmp 1777
---
> /tmp 775
23043,23044c23043,23044
< /var/log/journal 2755
< /var/log/journal/remote 2755
---
> /var/log/journal 755
> /var/log/journal/remote 755
23053,23054c23053,23054
< /var/spool/mail 1777
< /var/tmp 1777
---
> /var/spool/mail 775
> /var/tmp 775
[bebe:~/code/archimg-archlinux] master ±
```

- docker image `archimg-base-root`: self-built and calling download + build as root. For some reasons, I do believe It has to something with this. It's too late today to actually push something to master.

**Comment by bebehei** [comment]
@heichblatt Thank you for your observation. The symptoms are fixed in f61f490417e3419f1b5256bd4f16a42e126b1ae6.

But I let this Bugreport open, as I would also like to know, where the root cause is and fix it there.

Also if you would like to help me in this organisation to provide basic Archlinux images, I'd appreciate this.
